src/main/java/karuna/karuna_backend/visitor/message/domain/VisitorMessageRepository.java

```diff
@@ -1,9 +1,11 @@
 package karuna.karuna_backend.visitor.message.domain;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
 
 import java.util.List;
 
+@Repository
```

Comment:
Is unused when u extends jparepository

Reply:
according to discuss on discord I leave it as it is